src/ViewRendererBundle/Twig/Loader/ChameleonTwigLoader.php

```diff
@@ -39,10 +39,10 @@ private function init()
         $this->initialized = true;
     }
 
-    protected function findTemplate($name)
+    protected function findTemplate($name, $throw = true)
```

Comment:
Could you rebase the branch onto 6.2.x? I think it's better not to mix this one in.

Reply:
These "changes" are now removed - but still show up here...